CI/appveyor/build.sh.inc

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Options
+MAKE_J=-j8
+GCC_VERSION=7.3-win32
+
+# ------------------------------------------------------------------------------
+
+set -e
```

Comment:
it's a good idea to do tracing when setting `-e `
so, `set -e` -> `set -xe`  or just `set -x` on a new line;

this is good to have enabled until things are completely sorted/working;
`set -e` will silently fail and you won't always have debugging info